.buildkite/pipeline.yml

```diff
@@ -805,20 +805,35 @@ steps:
   - group: "GPU"
     steps:
 
-      - label: "GPU: GPU implicit baro wave"
+      - label: "GPU: GPU implicit dry baro wave"
```

Comment:
Can we just leave out the "dry" for the dry case, and add moist for the moist case? I think the naming scheme is easier to follow if we leave out default values (dry is the default)

Reply:
Sure we can! I will change it